tasks/broad/InternalArraysTasks.wdl

```diff
@@ -366,3 +366,36 @@ task GetNextArraysQcAnalysisVersionNumber {
     Int analysis_version_number = read_int(stdout())
   }
 }
+
+task ResolveExtendedIlluminaManifestFile {
+  input {
+    File extended_manifest_map_file
+    String bpm_filename
+    String egt_filename
```

Comment:
Is the cluster file being used other than the error messages?

Reply:
Not yet.  It probably should be used to help discriminate - good catch.  I was on the fence about it, but I think it's better to include it in the grep than not.